recipe-client-addon/lib/EventEmitter.jsm

```diff
@@ -20,15 +19,13 @@ this.EventEmitter = function(driver) {
       Promise.resolve()
         .then(() => {
           if (!(eventName in listeners)) {
-            driver.log(`EventEmitter: Event fired with no listeners: ${eventName}`);
+            log.info(`EventEmitter: Event fired with no listeners: ${eventName}`);
             return;
           }
-          // freeze event to prevent handlers from modifying it
-          const frozenEvent = Object.freeze(event);
```

Comment:
So I just checked in Firefox's browser console using something like this:

```js
let x = {foo: 5};
x = Object.freeze(x);
y = Cu.cloneInto(y, content) // this only works with e10s disabled
Object.isFrozen(y) // returns false
y.foo = 10; // changes y.foo
```

So I don't think this works as-is. :-(

If the event object is cloned into the sandbox, I wouldn't expect there to be issues with calling `Object.freeze` on it. What exactly happens if you try and freeze the object?